packages/@angular/cli/tasks/eject.ts

```diff
@@ -320,6 +320,9 @@ class JsonWebpackSerializer {
   serialize(config: any): string {
     // config = Object.assign({}, config);
     config['plugins'] = this._pluginsReplacer(config['plugins']);
+    config['devServer'] = {
+      'historyApiFallback': true
+    }; // routes using PathLocationStrategy break without this
```

Comment:
Move this comment over the config, start with a capital letter and add a period at the end. Otherwise LGTM